docs/DECLARING_DEPENDENCIES.md

```diff
@@ -173,7 +173,15 @@ you are using at least Gradle 4.6. To do this:
 - Add a dependency on the BOM for the library you depend on
 - Remove the version from the dependency declarations of the artifacts in that library
 
-For an example, see [gax-java#690](https://github.com/googleapis/gax-java/pull/690/files).
```

Comment:
This example somehow does not use "platform".
https://docs.gradle.org/current/userguide/platforms.html#sub:bom_import. So updating with a valid example:

Reply:
Is this maybe something that changed from 4.x to 5.x? or 5.x to 6.x?

also "However often BOMs are not only providing versions as recommendations, but also a way to override any other version found in the graph. You can enable this behavior by using the enforcedPlatform keyword, instead of platform, when importing the BOM:"

Reply:
Gradle documentation has "platform" in BOM section since its 5.0 release.
https://docs.gradle.org/5.0/userguide/managing_transitive_dependencies.html#sec:bom_import
4.6 release note does not have it. https://docs.gradle.org/4.6/release-notes.html#bom-import

Updated the example code snippet to use enforcedPlatform.